test/built-ins/Temporal/TimeZone/prototype/getPlainDateTimeFor/limits.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime.prototype.getplaindatetimefor
+description: Numeric bounds check
+features: [Temporal]
+includes: [temporalHelpers.js]
+---*/
+
+const min = Temporal.Instant.from('-271821-04-20T00:00Z');
```

Comment:
I'd prefer to use `new Temporal.Instant(-8_640_000_000_000_000_000_000n)` and `new Temporal.TimeZone("-23:59")` to avoid coupling with from() where it's not necessary.

Also I think since that test was originally written a few years ago, it's now possible to have `new Temporal.TimeZone("-23:59.999999999")` so it might be better to use that here. The purpose of this test is to show that the representable range of PlainDateTime is enough to cover any possible combination of Instant and TimeZone, so it seems better to me to use the most extreme value down to the nanosecond.